module3/preparation/Week4/UpdatingResources.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Updating Resources in MVC Preparation
+---
+
+# Pre-work for [Updating Resources](/module3/lessons/Week4/UpdatingResources)
+
+In our next lesson, we are going to be using our MvcMovies application to update Movie and Review records.  To prepare for this lesson, let's revisit:
+* [Updating Records in a Database](/module2/lessons/Week1/SQLCRUD/)
+* [HTTP Requests and Response Cycle](/module3/lessons/week1/HowTheWebWorks)
+
+## Updating Records in a Database
+
+Work through the [W3Schools SQL Update Tutorial](https://www.w3schools.com/sql/sql_update.asp); make sure to use the 'Try It Yourself' buttons and the 'Test Yourself With Exercises' section.  As you go, answer the following questions in your notebook:
+1. What do you need to know about a record in order to update it?
+1. What are some pitfalls you need to be aware of as you are updating records?
+
+## Diagram HTTP Request/Response
+
+In your notebook:
```

Comment:
Do we want them to make the diagram for a new movie here or start thinking about the diagram for an update? If it's review, I think it would be good to clarify this is reviewing and then during the lesson we will talk about how update would be different.

Reply:
I'll change 'revisit' above to 'review' and add some additional notes to make it more clear!